recipes/libsbml-plus-packages/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libsbml-plus-packages" %}
+{% set version = "5.20.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sbmlteam/libsbml/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a196cab964b0b41164d4118ef20523696510bbfd264a029df00091305a1af540
+  
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - zlib
+    - bzip2
+    - libxml2
+    
+  host:
+    - libxml2
+    - zlib
+    - bzip2
+    
+  run:
+    - libxml2
+    - bzip2
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libsbml.so  # [linux]
+
+about:
+  home: https://github.com/vincent-noel/libsbml-plus-packages
```

Comment:
Please use the correct URL

Reply:
I'm now pointing to the official libsbml page for the home, and I also modified the dev_url to point to the official libsbml repository.